README.md

```diff
@@ -1,2 +1,29 @@
-# pygenesys
-Python for Generating Energy Systems
+# Python for Generating Energy Systems (PyGenESys)
+
+This package can be used to generate input files for the [Temoa](https://github.com/temoaproject/temoa) modelling tool. Transparency and repeatability are integral to the design motivations for Temoa. PyGenesys takes this concept
+even further by reducing the lead time and programming background required to
+run Temoa.
+PyGenesys offers technology models with cost and production time series data.
+The basic model is based on the state of Illinois and its flagship university, the University of Illinois at Urbana-Champaign.
```

Comment:
```suggestion
The model is based on the state of Illinois and its flagship university, the University of Illinois at Urbana-Champaign.
```

Reply:
``pygenesys`` is a model design tool and thus isn't specific to a particular region or system. Perhaps "test" model would have been better. But that's still inaccurate. The README will become clearer as the code base develops.